PluginsAndFeatures/azure-toolkit-for-intellij/rider/test/kotlin/org/jetbrains/plugins/azure/functions/helpers/csharp/AzureFunctionsPsiHelperTest.kt

```diff
@@ -25,6 +25,7 @@ package org.jetbrains.plugins.azure.functions.helpers.csharp
 import com.jetbrains.rider.test.asserts.shouldBeFalse
 import org.testng.annotations.Test
 
+// REVIEW @sdubov: Integration tests exist for TimerTrigger completion etc. Should this class be removed?
```

Comment:
FYI, removed this test in the other PRs as we indeed have an integration test

Reply:
@maartenba, please drop this class. I'm not sure what this test should check currently.